eduvpn/util.py

```diff
@@ -19,16 +19,23 @@
 from eduvpn.metadata import Metadata
 from eduvpn.exceptions import EduvpnException
 
+from typing import Any, Optional, Tuple
+from pil import Image
```

Comment:
The tests fail online because of these missing imports

pil and typing. I'm actually now sure if we need/use pil? If so don't we use pillow? i have to check.

Also the typing module needs to be installed for the py2 containers.

Reply:
Could not find any other occurrence with “pillow” in all the other files. I assume it’s just the an interchangeable name. The doc in the code says this:
![image](https://user-images.githubusercontent.com/36157555/63028352-f4abd680-beae-11e9-85fd-49f6df4cfc1a.png)

Reply:
You should remove the `pil` import statement and wrap the type description in quotes (make it a string). This is a workaround so the types are still checked, but an import is not required.

Reply:
This will probably also fix most of the broken tests on travis. Please have a look at the failing tests there. Do you know how to run the test suite? You can also try to run the tests in all the docker containers manually.